### PR TITLE
Adds Reviews to Page

### PR DESCRIPTION
**Pull request by ubahl**
Allows users to add reviews to the page.

Adds some style aspects: CSS on the comments and in the white background. Adds a form to the page, which allows users to write reviews and add it to the page. Also adds the user's name.

Screenshot 2020-06-29 at 11 34 04 AM

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -30,22 +30,20 @@ async function getRandomMessage() {
...
-    document.getElementById('white-background').innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
-  }
-
-  /* alternate attempt which didn't work: 
-  *
   reviewBox = document.getElementById('white-background');
+
   for(var i = 0; i < message.length; i++) {
-    newComment = document.createElement('review-text');
+    // reviewBox.innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
```
Comment (nwertz) [inline portfolio/src/main/webapp/script.js:36]: Any reason to keep this commented out? If so, maybe just add another comment explaining if/when you will use this line later :)

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -30,22 +30,25 @@ async function getRandomMessage() {
 async function getReviewData() {
   const response = await fetch('/data');
   const message = await response.json();
+  reviewBox = document.getElementById('white-background');
 
   for(var i = 0; i < message.length; i++) {
-    document.getElementById('white-background').innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
-  }
+    // reviewBox.innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
```
Comment (mrboxtobox) [inline portfolio/src/main/webapp/script.js:36]: You can remove this.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -50,6 +50,8 @@ body {
     height: auto;
     margin: 50px auto;
     width: 60%;
+    padding-top: 2%;
```
Comment (nwertz) [inline portfolio/src/main/webapp/style.css:53]: nit: prefer alphabetizing css attributes

**Review by mrboxtobox — APPROVED** [review]
Nice!

File: portfolio/src/main/java/com/google/sps/servlets/Review.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+
+public class Review {
```
Comment (nwertz) [inline portfolio/src/main/java/com/google/sps/servlets/Review.java:3]: Nice! Can we add some class-level documentation/comments above this describing what the 'Review' class is/what it does?

**Review by nwertz — APPROVED** [review]

**Comment by ubahl** [comment]
thanks for all the great comments!!! it's very helpful :)

**Pull request merged**